docs/UserGuide.md

```diff
@@ -276,6 +291,107 @@ Format: `unlink s/STUDENT_INDEX c/CLASS_INDEX`
 Examples:
 *  `unlink s/1 c/2` Unlinks the 1st student from the 2nd class in the respective lists.
 
+### Managing Lessons
+
+#### Adding a lesson : `add-lesson`
+
+Adds a lesson to the application.
+
+Format: `add-lesson c/CLASS_INDEX d/DAY st/STARTTIME et/ENDTIME v/VENUE r/NO_OF_TIMES`
+
+Examples:
+* `add-lesson c/1 d/MONDAY st/8.00AM et/10.00AM v/COM1 #01-01 r/13`
+
+#### Editing a lesson : `edit-lesson`
+
+Edits an existing lesson in the application.
+
+Format: `edit-lesson c/CLASS_INDEX l/LESSON_INDEX [d/DAY] [st/STARTTIME] [et/ENDTIME] [v/VENUE]`
+
+* Edits the lesson at the specified `CLASS_INDEX` + `LESSON_INDEX`.
+* The indexes **must be positive integers** 1, 2, 3, …​
+* User has to specify at least 1 field (DAY/STARTTIME/ENDTIME/VENUE) to be changed.
+
+Examples:
+*  `edit-lesson c/1 l/1 d/TUESDAY st/10.00AM et/12.00PM v/COM2 #02-02` Edits the 1st lesson of the 1st class to be held on Tuesdays, 10.00AM to 12.00PM at COM2 #02-02.
+
+#### Deleting a lesson : `delete-lesson`
+
+Deletes the specified lesson from the application.
+
+Format: `delete-lesson c/CLASS_INDEX l/LESSON_INDEX`
+
+* Deletes the lesson at the specified `CLASS_INDEX` + `LESSON_INDEX`.
+* The indexes **must be positive integers** 1, 2, 3, …​
+
+### Managing Attendance Records
+
+#### Adding an attendance record : `add-attendance`
+
+Adds an attendance record to the application.
+
+Format: `add-attendance c/CLASS_INDEX l/LESSON_INDEX s/STUDENT_INDEX w/WEEK p/PARTICIPATION_SCORE`
+
+Examples:
+* `add-attendance c/1 l/1 s/1 w/1 p/1`
+
+#### Editing an attendance record : `edit-attendance`
+
+Edits an existing attendance record in the application.
+
+Format: `edit-attendance c/CLASS_INDEX l/LESSON_INDEX s/STUDENT_INDEX w/WEEK p/PARTICIPATION_SCORE`
+
+* Edits the participation score of the attendance record at the specified `CLASS_INDEX` + `LESSON_INDEX` + `STUDENT_INDEX` + `WEEK`.
+* The indexes **must be positive integers** 1, 2, 3, …​
+* `PARTICIPATION_SCORE` and `WEEK` must be non-negative integers.
+
+Examples:
+*  `edit-attendance m/1 l/1 s/1 w/1 p/10` Edits the participation score of the 1st student of the 1st lesson of the 1st class to 10 points.
```

Comment:
The week seems to be missing from the description, same for the subsequent attendance record commands.